test/calendar.ts

```diff
@@ -0,0 +1,97 @@
+import {assert} from 'mocha-webdriver';
+import {getGrist} from "./getGrist";
+
+//not to pretty way to get events from currently used calendar control. but it's working. 
+function getAbstractFromCalendarObject(eventId: number) {
+    return `
+            const calendarObject =  this.calendarHandler.calendar.getEvent(${eventId},"cal1"); 
+            if(calendarObject)
+            {
+                return{
+                    title: calendarObject?.title,
+                    startDate: calendarObject?.start.toString(),
+                    endDate: calendarObject?.end.toString(),
+                    isAllDay: calendarObject?.isAllday??false
+                }
+            }else{
+                return calendarObject
+            }`
+}
+
+describe('calendar', function () {
+    this.timeout(20000);
+    const grist = getGrist();
+
+    before(async function () {
+        const docId = await grist.upload('test/fixtures/docs/Calendar-UnitTest.grist');
```

Comment:
Maybe leave out the `-UnitTest` from the name, just so all the next files added don't stick it on as a tradition.